src/Exception.php

```diff
@@ -149,7 +149,7 @@ public function getColorfulText()
             $output .= "\n\033[0mCaused by Previous Exception:\n";
             $output .= "\033[1;31m".get_class($p).': '.$p->getMessage()."\033[0;31m".
                 ($p->getCode() ? ' [code: '.$p->getCode().']' : '');
-            if ($p instanceof \atk4\core\Exception) {
+            if ($p instanceof self) {
```

Comment:
i have checked with documentation and i think "self" is a correct way here. Basically it must use __CLASS__ or atk4/core/exception. If $p decends from it it's ok. 

If we use "static" than it would be the get_class($this), which could mean it's atk4\data\Exception and $p instanceof  will fail.

Reply:
i confirm behaviour of self over static. +1 for self

Reply:
ok